scripts/cy-wrapper.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+
+if test -z "$BASH_VERSION"; then
+  echo "Please run this script using bash, not sh or any other shell." >&2
+  exit 1
+fi
+
+export CY_API_URL="${CY_API_URL:-https://http-api.cycloid.io}"
+export CY_BINARIES_PATH="${CY_BINARIES_PATH:-$HOME/.cycloid}"
+
+# Ensure we have Cycloid directory created
+if ! [ -d "$CY_BINARIES_PATH" ]; then
+  mkdir -p $CY_BINARIES_PATH
+fi
+
+# If specified in the commandline use it
+if [[ `echo "$@" | grep -E --  "--api-url(=|\s+)([^\]+)"` ]]; then
+  CY_API_URL=$(echo "$@" | sed -r "s/.*--api-url(=|\s+)([^ ]+).*/\2/")
+fi
+
+# Remove trailing /
+CY_API_URL=${CY_API_URL%/}
```

Comment:
We don't need to perform this check, it seem to work even with a duplicate trailing `/`:

```shell
$ curl $CY_API_URL/version
{"data":{"branch":"develop","revision":"04910f0119a4ec6e7e6d311c778e10850f584cdc","version":"1.0.50-rc.4"}}
$ curl $CY_API_URL//version
{"data":{"branch":"develop","revision":"04910f0119a4ec6e7e6d311c778e10850f584cdc","version":"1.0.50-rc.4"}}
$ curl $CY_API_URL/////version
{"data":{"branch":"develop","revision":"04910f0119a4ec6e7e6d311c778e10850f584cdc","version":"1.0.50-rc.4"}}
```

Reply:
After discussion we will keep it just in case. It will also make error logs more cleaner avoiding to display some // url's